src/main.js

```diff
@@ -55,22 +59,30 @@ if (crashReporterDirectory) {
 		}
 	}
 
-	// Crashes are stored in the temp directory by default, so we
+	// Crashes are stored in the crashDumps directory by default, so we
 	// need to change that directory to the provided one
 	console.log(`Found --crash-reporter-directory argument. Setting temp directory to be '${crashReporterDirectory}'`);
-	app.setPath('temp', crashReporterDirectory);
-
-	// Start crash reporter
-	const { crashReporter } = require('electron');
-	const productName = (product.crashReporter && product.crashReporter.productName) || product.nameShort;
-	const companyName = (product.crashReporter && product.crashReporter.companyName) || 'Microsoft';
-	crashReporter.start({
-		companyName: companyName,
-		productName: process.env['VSCODE_DEV'] ? `${productName} Dev` : productName,
-		submitURL: '',
-		uploadToServer: false
-	});
+	app.setPath('crashDumps', crashReporterDirectory);
+}
+
+// Start crash reporter for all processes
+const productName = (product.crashReporter && product.crashReporter.productName) || product.nameShort;
```

Comment:
@deepak1556 I think by now you can write this and the next line as `product.crashReporter?.productName) || product.nameShort` (optional chaining).